src/clojurebot/core.clj

```diff
@@ -82,15 +91,27 @@
  (fn [msg]
    (str "=> " (pr-str (eval-string msg)))))
 
+; (reg-command
+;   "doc"
+;   (fn [sym-name]
+;     (if (not (clojure.string/blank? sym-name))
+;         (->> (symbol sym-name)
+;              (resolve)
+;              (meta)
+;              (doc->string)
+;              (replace-ws))
+;         "ERROR: No arguments provided")))
+
 (reg-command
   "doc"
   (fn [sym-name]
-    (if sym-name
-      (->> (symbol sym-name)
-           (resolve)
-           (meta)
-           (:doc))
-      (println "ERROR: No arguments provided"))))
+    (if (not (clojure.string/blank? sym-name))
+        (->> (if-let [sym (symbol sym-name)]
```

Comment:
This isn't correct. My comment said: 

```clojure
(if-let [sym (resolve (symbol sym-name))]
  (->> sym
       (meta)
       ...)
  (str "No docs for symbol:" sym-name))
```

Also, all forms in the `(->> ...)` should have the same indentation.